### PR TITLE
[FRONT] Calendar component

### PR DESCRIPTION
**Pull request by Anisospina**
## Summary

- Calendar component
- font Cera Pro

## Screenshots
calendar

## GitHub Card

#6

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -27,6 +27,7 @@ target 'woloxManagementOffice' do
   pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
   pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
 
+
```
Comment (andyOlarte514) [inline ios/Podfile]: revert this change ⚠️

File: src/app/components/CustomCalendar/index.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  state = {initialDate: null, finalDate: null, daysSelected: null};
+
+  handleClickModalFilter = day => {
+    const {initialDate, finalDate} = this.state;
+    if (!initialDate || finalDate || day.dateString < initialDate) {
+      this.setState(
+        {initialDate: day.dateString, finalDate: null, daysSelected: null},
+        this.handleInitialDate,
+      );
+    } else {
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/index.js]: you can delete this brackets

File: src/app/components/CustomCalendar/index.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        markingType={PERIOD}
+        theme={myTheme}
+        renderArrow={this.renderArrow}
+      />
+    );
+  }
+}
+
+CustomCalendar.propTypes = {
+  onAcceptDate: PropTypes.func,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/index.js]: isRequired

**Review by andyOlarte514 — CHANGES_REQUESTED** [review]
Nice, just some comments!!!

**Review by andyOlarte514 — APPROVED** [review]
👌

File: babel.config.js [hunk 4]
```diff
@@ -6,17 +6,19 @@ module.exports = {
...
-          '@utils': './src/utils'
-        }
-      }
-    ]
-  ]
+          '@utils': './src/utils',
+        },
+      },
+    ],
+  ],
```
Comment (mpujolWolox) [inline babel.config.js]: Delete `,`

File: index.d.ts [hunk 5]
```diff
@@ -0,0 +1,2 @@
+declare module "*.png"
```
Comment (mpujolWolox) [inline index.d.ts]: `;` ?

File: jest.config.tsx [hunk 6]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  preset: 'react-native',
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
```
Comment (mpujolWolox) [inline jest.config.tsx]: Delete `,`

File: src/app/components/CustomCalendar/constants.tsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    i18next.t('calendar:march'),
+    i18next.t('calendar:april'),
+    i18next.t('calendar:may'),
+    i18next.t('calendar:june'),
+    i18next.t('calendar:july'),
+    i18next.t('calendar:august'),
+    i18next.t('calendar:september'),
+    i18next.t('calendar:october'),
+    i18next.t('calendar:november'),
+    i18next.t('calendar:december'),
```
Comment (mpujolWolox) [inline src/app/components/CustomCalendar/constants.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+      backgroundColor: white,
+      paddingHorizontal: refRatioScale(20),
+      borderBottomLeftRadius: 10,
+      borderBottomRightRadius: 10,
+      paddingVertical: refRatioScale(15),
+    },
+  },
+  'stylesheet.calendar.header': {
+    week: {
+      marginTop: 3,
```
Comment (josepplloo) [inline src/app/components/CustomCalendar/styles.tsx]: don't use odd values

**Review by josepplloo — CHANGES_REQUESTED** [review]
Waiting for new changes

**Review by josepplloo — APPROVED** [review]
After you complete the changes in the constants' names you can merge :)

**Review by mpujolWolox — DISMISSED** [review]

File: .gitignore [hunk 9]
```diff
@@ -61,4 +61,5 @@ buck-out/
 package-lock.json
 yarn.lock
 Gemfile.lock
-Podfile.lock
\ No newline at end of file
+Podfile.lock
+tsconfig.json
```
Comment (andyOlarte514) [inline .gitignore]: If you don't upload this file, how will we have it?

File: babel.config.js (hunk 4 above)
Comment (andyOlarte514) [reply babel.config.js]: Remove `,`

File: jest.config.tsx (hunk 6 above)
Comment (andyOlarte514) [reply jest.config.tsx]: same

File: src/app/components/CustomCalendar/constants.tsx [hunk 10]
```diff
@@ -0,0 +1,12 @@
+import {getMonthNames, getDayLetter} from './utils';
+
+export const CONFIG_CALENDAR = {
+  monthNames: getMonthNames(),
+  dayNamesShort: getDayLetter(),
+};
+
+export const VARIABLES = {
+  left: 'left',
+  period: 'period',
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/constants.tsx]: same delete `,`

File: src/app/components/CustomCalendar/constants.tsx [hunk 11]
```diff
@@ -0,0 +1,12 @@
+import {getMonthNames, getDayLetter} from './utils';
+
+export const CONFIG_CALENDAR = {
+  monthNames: getMonthNames(),
+  dayNamesShort: getDayLetter(),
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/constants.tsx]: same delete` ,`

File: src/app/components/CustomCalendar/index.tsx [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+import {myTheme} from './styles';
+
+LocaleConfig.locales[DEFAULT_LOCALE] = CONFIG_CALENDAR;
+LocaleConfig.defaultLocale = DEFAULT_LOCALE;
+
+class CustomCalendar extends Component<Props, CalendarState> {
+  state = {
+    initialDate: null,
+    finalDate: null,
+    daysSelected: null,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/index.tsx]: same delete `,`

File: src/app/components/CustomCalendar/index.tsx [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+  };
+
+  handleClickModalFilter = day => {
+    const {initialDate, finalDate} = this.state;
+    if (!initialDate || finalDate || day.dateString < initialDate) {
+      this.setState(
+        {initialDate: day.dateString, finalDate: null, daysSelected: null},
+        this.handleInitialDate,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/index.tsx]: same

File: src/app/components/CustomCalendar/index.tsx [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+
+  handleInitialDate = () => {
+    const {initialDate} = this.state;
+    const daysSelected = {
+      [initialDate]: {
+        startingDay: true,
+        endingDay: true,
+        selected: true,
+        color: MALIBU,
+        textColor: WHITE,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/index.tsx]: same

File: src/app/components/CustomCalendar/index.tsx [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+      },
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/index.tsx]: same

File: src/app/components/CustomCalendar/index.tsx [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+  };
+
+  handleSelectRange = () => {
+    const {initialDate, finalDate} = this.state;
+    const {onAcceptDate} = this.props;
+    onAcceptDate({initialDate, finalDate});
+  };
+
+  renderArrow = direction => {
+    return (
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/index.tsx]: `return` is not necessary use

```
renderArrow = direction => (
       <Image
         style={{tintColor: BOMBAY}}
         resizeMode="contain"
         source={direction === VARIABLES.left ? prevImg : nextImg}
       />
     );
```

File: src/app/components/CustomCalendar/styles.tsx [hunk 17]
```diff
@@ -0,0 +1,94 @@
+import {
+  MIRAGE,
+  CATEDBLUE,
+  GALLERY,
+  WHITE,
+  BOMBAY,
+  OUTERSPACE,
+  CERULEAN,
+  TRANSPARENT,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: delete `,`

File: src/app/components/CustomCalendar/styles.tsx [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+} from '@constants/colors';
+import {refRatioScale, isIos} from '@constants/platform';
+import {
+  MARGIN_HALF_PADDING,
+  MARGIN_PADDING_12,
+  MARGIN_PADDING_TRIPLE,
+  MARGIN_PADDING,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 19]
```diff
@@ -0,0 +1,94 @@
...
+
+export const myTheme = {
+  calendarBackground: TRANSPARENT,
+  textDisabledColor: BOMBAY,
+  todayTextColor: CERULEAN,
+  'stylesheet.day.period': {
+    wrapper: {
+      alignItems: 'center',
+      alignSelf: 'stretch',
+      paddingBottom: refRatioScale(10),
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 20]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    text: {
+      marginTop: isIos ? MARGIN_PADDING : MARGIN_HALF_PADDING,
+      fontSize: fontSizes.SMALL,
+      fontFamily: fontCera.REGULAR,
+      fontWeight: fontWeights.NORMAL,
+      color: MIRAGE,
+      marginBottom: ZERO,
+      paddingBottom: isIos ? ZERO : MARGIN_HALF_PADDING,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 21]
```diff
@@ -0,0 +1,94 @@
...
+    },
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 22]
```diff
@@ -0,0 +1,94 @@
...
+  },
+  'stylesheet.calendar.main': {
+    week: {
+      marginVertical: ZERO,
+      flexDirection: 'row',
+      justifyContent: 'space-around',
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 23]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    monthView: {
+      backgroundColor: WHITE,
+      paddingHorizontal: refRatioScale(20),
+      borderBottomLeftRadius: 10,
+      borderBottomRightRadius: 10,
+      paddingVertical: refRatioScale(15),
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same :sad:

File: src/app/components/CustomCalendar/styles.tsx [hunk 24]
```diff
@@ -0,0 +1,94 @@
...
+    },
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 25]
```diff
@@ -0,0 +1,94 @@
...
+      marginTop: 3,
+      flexDirection: 'row',
+      justifyContent: 'space-around',
+      borderBottomWidth: 1,
+      borderBottomColor: GALLERY,
+      paddingHorizontal: refRatioScale(20),
+      paddingVertical: refRatioScale(5),
+      backgroundColor: WHITE,
+      borderTopLeftRadius: 10,
+      borderTopRightRadius: 10,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 26]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    dayHeader: {
+      marginVertical: MARGIN_PADDING_12,
+      width: SIZE_CIRCLE,
+      color: CATEDBLUE,
+      fontSize: fontSizes.XSMALL,
+      textAlign: 'center',
+      fontFamily: fontCera.REGULAR,
+      fontWeight: fontWeights.SEMI_BOLD,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 27]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    monthText: {
+      fontSize: fontSizes.MEDIUM,
+      fontFamily: fontCera.REGULAR,
+      fontWeight: fontWeights.SEMI_BOLD,
+      color: OUTERSPACE,
+      margin: ZERO,
+      padding: ZERO,
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: only add commas at the end if you then add another value or property

File: src/app/components/CustomCalendar/styles.tsx [hunk 28]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    header: {
+      flexDirection: 'row',
+      justifyContent: 'space-around',
+      paddingVertical: refRatioScale(5),
+      alignItems: 'center',
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/utils.tsx [hunk 29]
```diff
@@ -0,0 +1,25 @@
+import {monthName, dayWeekName} from '@utils/date';
+import {capitalize} from '@utils/stringsUtils';
+
+export const getMonthNames = () => {
+  const month = [];
+
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/utils.tsx]: delete space

File: src/app/components/CustomCalendar/utils.tsx [hunk 30]
```diff
@@ -0,0 +1,25 @@
...
+  for (let i = 1; i <= 12; i += 1) {
+    month.push(capitalize(monthName(i)));
+  }
+  return month;
+};
+
+export const getDayLetter = () => {
+  const days = [];
+
```
Comment (andyOlarte514) [inline src/app/components/CustomCalendar/utils.tsx]: delete space

File: src/utils/stringsUtils.tsx [hunk 31]
```diff
@@ -0,0 +1,3 @@
+export const capitalize = str => {
+  return str.charAt(0).toUpperCase() + str.slice(1);
```
Comment (andyOlarte514) [inline src/utils/stringsUtils.tsx]: `{}` and `return` not necessary

**Review by andyOlarte514 — CHANGES_REQUESTED** [review]
Review the commas and only add the commas at the end if you then add another value or property

**Review by andyOlarte514 — APPROVED** [review]
🎉

File: src/app/components/CustomCalendar/constants.tsx [hunk 32]
```diff
@@ -0,0 +1,12 @@
+import {getMonthNames, getDayLetter} from './utils';
+
+export const CONFIG_CALENDAR = {
+  monthNames: getMonthNames(),
+  dayNamesShort: getDayLetter()
+};
+
+export const VARIABLES = {
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/constants.tsx]: use more descriptive name

File: src/app/components/CustomCalendar/constants.tsx [hunk 33]
```diff
@@ -0,0 +1,12 @@
...
+  left: 'left',
+  period: 'period'
+};
+export const PERIOD = 'period';
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/constants.tsx]: you are repeating this constant

File: src/app/components/CustomCalendar/index.tsx [hunk 34]
```diff
@@ -0,0 +1,92 @@
...
+LocaleConfig.defaultLocale = DEFAULT_LOCALE;
+
+class CustomCalendar extends Component<Props, CalendarState> {
+  state = {
+    initialDate: null,
+    finalDate: null,
+    daysSelected: null
+  };
+
+  handleClickModalFilter = day => {
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/index.tsx]: why you propose typescript if you don't add type to the params?

File: src/app/components/CustomCalendar/styles.tsx [hunk 35]
```diff
@@ -0,0 +1,94 @@
...
+  'stylesheet.calendar.main': {
+    week: {
+      marginVertical: ZERO,
+      flexDirection: 'row',
+      justifyContent: 'space-around'
+    },
+    monthView: {
+      backgroundColor: WHITE,
+      paddingHorizontal: refRatioScale(20),
+      borderBottomLeftRadius: 10,
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/styles.tsx]: why no use refRatioScale(10)? and extract this to a constant

File: src/app/components/CustomCalendar/styles.tsx [hunk 36]
```diff
@@ -0,0 +1,94 @@
...
+      borderBottomRightRadius: 10,
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 37]
```diff
@@ -0,0 +1,94 @@
...
+    week: {
+      marginTop: 3,
+      flexDirection: 'row',
+      justifyContent: 'space-around',
+      borderBottomWidth: 1,
+      borderBottomColor: GALLERY,
+      paddingHorizontal: refRatioScale(20),
+      paddingVertical: refRatioScale(5),
+      backgroundColor: WHITE,
+      borderTopLeftRadius: 10,
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 38]
```diff
@@ -0,0 +1,94 @@
...
+      borderTopRightRadius: 10
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/styles.tsx]: same

File: src/app/components/CustomCalendar/styles.tsx [hunk 39]
```diff
@@ -0,0 +1,94 @@
...
+      fontFamily: fontCera.REGULAR,
+      fontWeight: fontWeights.SEMI_BOLD,
+      color: OUTERSPACE,
+      margin: ZERO,
+      padding: ZERO
+    },
+    header: {
+      flexDirection: 'row',
+      justifyContent: 'space-around',
+      paddingVertical: refRatioScale(5),
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/styles.tsx]: you have `refRatioScale(5)` repeat, extract to a constant

File: src/app/screens/Login/styles.js [hunk 40]
```diff
@@ -1,28 +1,27 @@
...
-    borderBottomWidth: 1
+    borderBottomWidth: 1,
   },
   formElement: {
     padding: 3,
-    margin: 5,
-    backgroundColor: transparent,
     height: 30,
-    width: 200
+    width: 200,
```
Comment (jeymarsonwolox) [inline src/app/screens/Login/styles.js]: `refRatioScale(200)`

File: src/app/screens/Login/styles.js [hunk 41]
```diff
@@ -1,28 +1,27 @@
...
   },
   formButton: {
     backgroundColor: green,
     padding: 10,
```
Comment (jeymarsonwolox) [inline src/app/screens/Login/styles.js]: same

File: src/app/screens/Login/styles.js [hunk 42]
```diff
@@ -1,28 +1,27 @@
...
     borderRadius: 3,
-    margin: 20
-  }
+    margin: 20,
```
Comment (jeymarsonwolox) [inline src/app/screens/Login/styles.js]: same

File: src/constants/dimensions.tsx [hunk 43]
```diff
@@ -0,0 +1,26 @@
...
+export const MARGIN_PADDING_TRIPLE = refRatioScale(30);
+export const MARGIN_PADDING_FOURTH = refRatioScale(40);
+export const MARGIN_PADDING_FIFTH = refRatioScale(50);
+export const MARGIN_PADDING_SIX = refRatioScale(60);
+export const MARGIN_PADDING_SEVEN = refRatioScale(70);
+export const MARGIN_PADDING_EIGHTH = refRatioScale(80);
+export const MARGIN_PADDING_NINE = refRatioScale(90);
+export const MARGIN_PADDING_TEN = refRatioScale(100);
+export const MARGIN_PADDING_150 = refRatioScale(150);
+export const MARGIN_PADDING_250 = refRatioScale(250);
```
Comment (jeymarsonwolox) [inline src/constants/dimensions.tsx:22]: why don't you use these constants in the other files?

**Review by jeymarsonwolox — CHANGES_REQUESTED** [review]
Great!! just a few comments

File: src/app/components/CustomCalendar/styles.tsx [hunk 44]
```diff
@@ -0,0 +1,96 @@
...
+  MARGIN_PADDING_DOUBLE,
+  MARGIN_PADDING_15,
+  MARGIN_PADDING_12,
+  MARGIN_PADDING_TRIPLE,
+  MARGIN_PADDING
+} from '@constants/dimensions';
+import {fontSizes, fontWeights, fontCera} from '@constants/fonts';
+
+const SIZE_CIRCLE = isIos ? refRatioScale(34) : MARGIN_PADDING_TRIPLE;
+const ZERO = refRatioScale(0);
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/styles.tsx]: this is no necesary, `const ZERO = 0;`

File: src/app/components/CustomCalendar/styles.tsx [hunk 45]
```diff
@@ -0,0 +1,96 @@
...
+      backgroundColor: WHITE,
+      paddingHorizontal: MARGIN_PADDING_DOUBLE,
+      borderBottomLeftRadius: MARGIN_PADDING,
+      borderBottomRightRadius: MARGIN_PADDING,
+      paddingVertical: MARGIN_PADDING_15
+    }
+  },
+  'stylesheet.calendar.header': {
+    week: {
+      marginTop: 3,
```
Comment (jeymarsonwolox) [inline src/app/components/CustomCalendar/styles.tsx]: `refRatioScale(3)` ?

File: src/app/components/CustomTextInput/index.js [hunk 46]
```diff
@@ -73,7 +73,7 @@ CustomTextInput.defaultProps = {
   keyboardType: 'default',
   multiline: false,
   bottomBorder: false,
-  maxHeight: 200
+  maxHeight: 200,
```
Comment (jeymarsonwolox) [inline src/app/components/CustomTextInput/index.js:76]: add `refRatioScale(200)` in `src/constants/dimensions.tsx` and import here

File: src/app/screens/Login/styles.js [hunk 47]
```diff
@@ -14,15 +17,13 @@ export default StyleSheet.create({
   },
   formElement: {
     padding: 3,
-    margin: 5,
-    backgroundColor: transparent,
-    height: 30,
-    width: 200
+    height: refRatioScale(30),
+    width: refRatioScale(200)
```
Comment (jeymarsonwolox) [inline src/app/screens/Login/styles.js]: add `refRatioScale(200)` in `src/constants/dimensions.tsx` and import here

File: src/app/screens/Login/styles.js [hunk 48]
```diff
@@ -14,15 +17,13 @@ export default StyleSheet.create({
   },
   formElement: {
     padding: 3,
-    margin: 5,
-    backgroundColor: transparent,
-    height: 30,
-    width: 200
+    height: refRatioScale(30),
```
Comment (jeymarsonwolox) [inline src/app/screens/Login/styles.js]: use `MARGIN_PADDING_TRIPLE `

**Review by jeymarsonwolox — CHANGES_REQUESTED** [review]
Excellent, a few more comments

File: src/app/components/CustomTextInput/index.js (hunk 46 above)
Comment (Anisospina) [reply src/app/components/CustomTextInput/index.js:76]: This file is modified because `the prettier` had a wrong configuration.

Comment (Anisospina) [reply src/app/components/CustomTextInput/index.js:76]: this moment ignore this file

File: src/constants/dimensions.tsx [hunk 49]
```diff
@@ -0,0 +1,27 @@
...
+
+export const GIF_SIZE = refRatioScale(290);
+export const IMG_SIZE = IS_SMALL_DEVICE ? refRatioScale(200) : MARGIN_PADDING_250;
+export const ICON_SIZE = MARGIN_PADDING_DOUBLE;
+export const EXTRA_HEIGHT_FORMS = MARGIN_PADDING;
+export const FORM_WIDTH = refRatioScale(200);
```
Comment (jeymarsonwolox) [inline src/constants/dimensions.tsx]: add `export const MARGIN_PADDING_200 = refRatioScale(200);`
and use `export const FORM_WIDTH = MARGIN_PADDING_200;`

**Review by jeymarsonwolox — APPROVED** [review]
Excellent!!

File: src/constants/dimensions.tsx (hunk 49 above)
Comment (Anisospina) [reply src/constants/dimensions.tsx]: 💔
